types/type.go

```diff
@@ -3,14 +3,15 @@ package types
 type NotificationType string
 
 const (
-	WorkitemCreate       NotificationType = "workitem.create"
-	WorkitemUpdate       NotificationType = "workitem.update"
-	CommentCreate        NotificationType = "comment.create"
-	CommentUpdate        NotificationType = "comment.update"
-	UserEmailUpdate      NotificationType = "user.email.update"
-	InvitationTeamNoorg  NotificationType = "invitation.team.noorg"
-	InvitationSpaceNoorg NotificationType = "invitation.space.noorg"
-	AnalyticsNotifyCVE   NotificationType = "analytics.notify.cve"
+	WorkitemCreate       	 NotificationType = "workitem.create"
+	WorkitemUpdate       	 NotificationType = "workitem.update"
+	CommentCreate        	 NotificationType = "comment.create"
+	CommentUpdate        	 NotificationType = "comment.update"
+	UserEmailUpdate      	 NotificationType = "user.email.update"
+	InvitationTeamNoorg  	 NotificationType = "invitation.team.noorg"
+	InvitationSpaceNoorg 	 NotificationType = "invitation.space.noorg"
+	AnalyticsNotifyCVE   	 NotificationType = "analytics.notify.cve"
+	AnalyticsNotifyVersion   NotificationType = "analytics.notify.version"
```

Comment:
For new notification_type, you need to register at [link](https://github.com/fabric8-services/fabric8-notification/blob/master/main.go#L114)

Reply:
It would be good, if you can test end-to-end flow by invoking `/api/notify` endpoint [link](https://github.com/fabric8-services/fabric8-notification/blob/master/controller/notify.go#L33).  You can run notification_service locally by making few changes in main.go file.  Contact me in case you need any help on running notification_service locally.